packages/api/src/publish.ts

```diff
@@ -196,10 +196,10 @@ export function publishPackage(storage: Storage): any {
         requestOptions: {
           host: req.hostname,
           protocol: req.protocol,
-          // @ts-ignore
-          headers: req.headers,
+          headers: req.headers as { [key: string]: string },
           username,
         },
+        uplinksLook: false,
```

Comment:
LGTM so far, just checking what's the side effects of this one, seems not even used inside, maybe is not even mandatory.